package.json

```diff
@@ -188,42 +188,10 @@
     "storybook-addon-specifications": "^1.0.15",
     "stylelint": "^7.3.1",
     "stylelint-config-standard": "^13.0.1",
-    "sugarss": "^0.1.6",
+    "sugarss": "^0.2.0",
     "surge": "^0.18.0",
     "webpack": "2.1.0-beta.25",
     "webpack-dev-middleware": "^1.8.1",
     "webpack-hot-middleware": "^2.11.0"
-  },
-  "jest": {
-    "scriptPreprocessor": "<rootDir>/tests/preprocessor",
-    "globals": {
-      "__DEV__": true,
-      "__PROD__": false,
-      "__DEBUG__": false,
-      "__COVERAGE__": false,
-      "__PROTOCOL__": false,
-      "__HOST__": false,
-      "__PORT__": false
-    },
-    "moduleFileExtensions": [
-      "js",
-      "jsx",
-      "json",
-      "css",
-      "sss",
-      "md"
-    ],
-    "moduleDirectories": [
-      "node_modules",
-      "universal",
-      "client"
-    ],
-    "moduleNameMapper": {
-      "^.+\\.(md|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/tests/__mocks__/fileMock.js",
-      "^.+\\.(css|less|scss)$": "identity-obj-proxy"
-    },
-    "testPathIgnorePatterns": [
-      "(/blueprints/.*|\\.md$)"
-    ]
```

Comment:
Yes it was a duplicate config i forgot to remove. The real configs are in `tests/configs` not `package.json`.